skt/kerneltree.py

```diff
@@ -274,6 +284,16 @@ def __get_remote_name(self, uri):
         return remote_name
 
     def merge_git_ref(self, uri, ref="master"):
+        """
+        Merge the patch from source kernel Git reference.
+
+        Args:
+            uri: The source kernel Git repo URL.
```

Comment:
This should mention that the URL points to the repo containing the branch/reference to merge, to distinguish form the repo which it is being merged into